modules/porous_flow/src/bcs/PorousFlowSinkBC.C

```diff
@@ -0,0 +1,42 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "PorousFlowSinkBC.h"
+
+registerMooseObject("PorousFlowApp", PorousFlowSinkBC);
```

Comment:
This guy needs markdown documentation linking to s11_act.i so users understand how to use it.

Reply:
Yeah, CIVET told me :-)

Reply:
added doco page linking `s11_act.i` to show an example